src/containers.h

```diff
@@ -0,0 +1,51 @@
+/* Copyright (C) 2021 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+#ifndef __CONTAINERS_H__
```

Comment:
I'm not a fan of this very generic `Container` naming for something that is quick specific. Would suggest `RangeFile` or `ByteRangeFiles` or something along those lines so it is more descriptive.

Reply:
Indeed, I was hesitating between genericity and specificity...
Should I just keep things specific and not think about containers for other keys/values ?